### DISCUSSION TITLE
Google Chrome 2011-2015 Logo for macOS, Please! 🙏

### DISCUSSION THREAD

**Discussion by shiro6x** [elrumo/macOS_Big_Sur_icons_replacements#1505] [Icon request]
Can someone please make a Chrome 2011-2015 logo for macOS? 

Please make it look like this one: 
![Chromemacosbigsur](https://github.com/user-attachments/assets/5113ead3-583c-4aad-a76d-76dc7bc3a0cb)

but replace it with the 2011-2015 one:
![chrome20112015](https://github.com/user-attachments/assets/43bce851-1038-4064-b6d6-0f9ce5c0dce3)

**Comment by futurejohn** [comment]
Here you go! 

This is the .icns file:
https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/a86e0007e4e06b7b2f28193d84f36ed2_1726771341276.icns

And here's a .png:
![chrome_classic_2011](https://github.com/user-attachments/assets/5975c0c9-a8a5-4ba1-9f28-631ca50c621c)
